xenalib/XenaPort.py

```diff
@@ -107,68 +107,68 @@ def __parse_stats(self, stats_list):
         storage = {}
         for line in stats_list:
             parms = line.split()
-            if parms[1] == 'PT_TOTAL':
```

Comment:
Same comment as before, if we have a way to tell python that all strings are encoded to avoid having 'encode()' called repeatedly

Reply:
Please give it a read : https://anonbadger.wordpress.com/2015/06/16/why-sys-setdefaultencoding-will-break-code/